packages/components/src/RadarChart/RadarChart.scss

```diff
@@ -0,0 +1,5 @@
+.tc-radar-chart {
+  :global(.recharts-polar-grid-concentric-polygon:not(:last-child)) {
```

Comment:
Just for info, what is this ? I see in storybook that there is a polygon if this is removed, but why is there this polygon in rechart ?

Reply:
Recharts writes a varying number of stepped polygons to a radar chart by default. Our UX design calls for only the last, outer polygon which shapes the overall chart. For now this just hides the excess inner polygons in an inexpensive way. It could possibly be refactored as an enhancement in the future.